content/cpi-api-v2-method/create-vm.md

```diff
@@ -16,7 +16,7 @@ As of V2 of the CPI API contract, `create_vm` returns an array of the resultant
  * `cloud_properties` [Hash]: Cloud properties hash specified in the deployment manifest under the VM's resource pool.
  * `networks` [Hash]: Networks hash that specifies which VM networks must be configured.
  * `disk_cids` [Array of strings] Array of disk cloud IDs for the disks that the created VM will most _likely_ attach. The disk cloud IDs could be used to optimize VM placement so that disks are located nearby.
- * `environment` [Hash]: Resource pool's env hash specified in the deployment manifest, including initial properties added by the BOSH director as shown below. Additionally, the director will append the following guaranteed values:
+ * `environment` [Hash]: Resource pool's env hash specified in the deployment manifest, including initial properties added by the BOSH director as shown below. It gets passed by the CPI to the agent where it can be found in the user data's `env` hash in `/var/vcap/bosh/settings.json`. Additionally, the director will append the following guaranteed values:
```

Comment:
Is it worth documenting the `settings.json` piece? Technically that's something the agent controls and could change. Just curious if/how this information is useful to folks.

Reply:
Oh, I like the idea of updating that agent-api-interactions docs with this sort of information. Also looks like it needs some attention since the v2 stuff. Feel free to add as much detail as you're interested in, otherwise create an issue if you want to delegate more thorough updates :)